bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml

```diff
@@ -1037,7 +1108,4 @@ spec:
   maturity: stable
   provider:
     name: IBM
-  relatedImages:
```

Comment:
This should be retained for Airgap (offline install) support. It comes from the [config/manifests/bases/ibm-websphere-liberty.clusterserviceversion.yaml](https://github.com/WASdev/websphere-liberty-operator/pull/389/files#diff-d16ec231c96d3d888f885fe8d75d7351ea35a69f4e480302f1f4945f6adec35b).